patches/super-three+0.137.0.patch

```diff
@@ -0,0 +1,50 @@
+diff --git a/node_modules/super-three/build/three.js b/node_modules/super-three/build/three.js
+index 8b13338..0de8780 100644
+--- a/node_modules/super-three/build/three.js
++++ b/node_modules/super-three/build/three.js
+@@ -18770,7 +18770,17 @@
+ 
+ 				const cameras = cameraVR.cameras;
+ 				var object = poseTarget || camera;
+-				const parent = object.parent;
++
++				// Object is the Group Three.js node that contains the virtual camera.  In 1.3.0, the
++				// original code for getting the parent is:
++				//   const parent = object.parent;
++				// In 1.2.0, the code for setting the parent local variable is:
++				//   const parent = camera.parent
++				// Therefore, in 1.3.0 the parent was the scene, where as in 1.2.0 the parent is the Group
++				// Three.js node.  We want to continue the 1.2.0 behavior in order for xr-aframe to work
++				// with both 1.2.0 and 1.3.0.
++				const parent = camera.parent
```

Comment:
nit: the surrounding style adds `;` on end of lines, so we should probably do that too, to be consistent.

Reply:
True, updated.